.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -13,20 +13,15 @@ variables:
   image: $BENCHMARKS_CI_IMAGE
   needs: ["setup_agent_version"]
   script:
-    - export ARTIFACTS_DIR="$(pwd)/reports" && (mkdir "${ARTIFACTS_DIR}" || :)
-    - export DD_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.dd_api_key --with-decryption --query "Parameter.Value" --out text)
     - git clone --branch trace-agent https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform /platform && cd /platform
-    - ./steps/capture-hardware-software-info.sh
     - ./steps/run-benchmarks.sh
-    - "./steps/upload-results-to-s3.sh || :"
   artifacts:
     name: "reports"
```

Comment:
Reference: https://github.com/DataDog/dd-trace-rb/blob/master/.gitlab/benchmarks.yml

For artifacts uploading, please use

```
artifacts:
    name: "artifacts"
    when: always
    paths:
      - platform/artifacts/
    expire_in: 3 months
```